includes/reader-revenue/my-account/class-woocommerce-my-account.php

```diff
@@ -234,7 +304,14 @@ public static function remove_required_fields( $required_fields ) {
 	public static function wc_get_template( $template, $template_name ) {
 		if ( 'myaccount/form-edit-account.php' === $template_name ) {
 			global $newspack_reset_password_arg;
-			$newspack_reset_password_arg = self::RESET_PASSWORD_URL_PARAM;
+			global $newspack_send_magic_link_arg;
+			$newspack_reset_password_arg  = self::RESET_PASSWORD_URL_PARAM;
+			$newspack_send_magic_link_arg = self::SEND_MAGIC_LINK_PARAM;
```

Comment:
Even though this has already been done for the password reset, why are we using globals here? `\Newspack\WooCommerce_My_Account::RESET_PASSWORD_URL_PARAM` is a public constant.

Reply:
I couldn't see a good reason for the globals, so I refactored in 975c28d to use the constants for both templates. I also fixed the URLs implemented in 723a5cd so that clicking these verify buttons redirects to the proper "My Account" page.